src/components/layout/layout.js

```diff
@@ -37,6 +38,10 @@ const Main = styled.main`
 
 const has_dataLayer = isBrowser() && window.dataLayer
 const cookie_expires = 7
+Cookies.defaults = {
```

Comment:
any reason why we put on layout?, imo, we should put it on storage.js, there is a CookieStorage configuration there

Reply:
As we're using `Cookies` here to setup cookies. I didn't find `CookieStorage` usage to set/get cookies here.

Reply:
we used cookies here only for `location` cookies, the setup is done on `common/storage.js`, we also used cookies in several locations, such as `traffic-source` for affiliate token cookies, login.js for getting oauth cookie, etc.